builtInFunctions/saveUserName.go

```diff
@@ -52,53 +49,70 @@ func (s *saveUserName) SetNewGasConfig(gasCost *vmcommon.GasCost) {
 	s.mutExecution.Unlock()
 }
 
-// ProcessBuiltinFunction sets the username to the account if it is allowed
-func (s *saveUserName) ProcessBuiltinFunction(
-	_, acntDst vmcommon.UserAccountHandler,
+func inputCheckForUserNameCall(
```

Comment:
[comment]

Just a remark, nothing to be done here specifically.
You could've make this func part of `(s *saveUserName)` in order not to pass extra args: `mapDnsAddresses` and `gasCost`

Reply:
no. as it is used in another component as well.